frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -335,30 +333,40 @@ function UpdatePollsDirectlyButton({
 
   const action = getPollsTransitionAction(pollsTransition);
   const reportTitle = getPollsReportTitle(pollsTransition);
+  const suggestVxScanText = (() => {
+    switch (pollsTransition) {
+      case 'open_polls':
+        return `Open polls on VxScan to save the ${reportTitle.toLowerCase()} before opening polls on VxMark.`;
+      case 'pause_voting':
+        return `Pause voting on VxScan to save the ${reportTitle.toLowerCase()} before pausing voting on VxMark.`;
+      case 'resume_voting':
+        return `Resume voting on VxScan to save the ${reportTitle.toLowerCase()} before resuming voting on VxMark.`;
+      case 'close_polls':
+        return `Close polls on VxScan to save the ${reportTitle.toLowerCase()} before closing polls on VxMark.`;
+      /* istanbul ignore next */
+      default:
+        throwIllegalValue(pollsTransition);
+    }
+  })();
 
   return (
     <React.Fragment>
-      <Button onPress={() => setIsConfirmationModalOpen(true)}>
-        {action} Polls
-      </Button>
+      <Button onPress={() => setIsConfirmationModalOpen(true)}>{action}</Button>
       {isConfirmationModalOpen && (
         <Modal
           centerContent
           content={
             <Prose textCenter id="modalaudiofocus">
               <h1>No {reportTitle} on Card</h1>
-              <p>
-                {action} polls on VxScan to save the {reportTitle.toLowerCase()}{' '}
-                before closing polls on VxMark.
-              </p>
+              <p>{suggestVxScanText}</p>
             </Prose>
           }
           actions={
             <React.Fragment>
+              <Button onPress={confirmUpdate}>{action} on VxMark Now</Button>
               <Button primary onPress={closeModal}>
                 Cancel
               </Button>
-              <Button onPress={confirmUpdate}>{action} VxMark Now</Button>
```

Comment:
@beausmith, with the longer button text, the buttons no longer fit on one row together. But if you reverse their order, they do stay on one line... not sure why that would be. I just switched their order because that was easier, but can go back and work on getting them on one line if this violates a design pattern.

![Screen Shot 2022-11-16 at 5 06 49 PM](https://user-images.githubusercontent.com/37960853/202329036-cbb37340-23d6-4096-94cd-79eed57bc081.png)

Reply:
This is in the modal where you change polls on VxMark directly, without using the card.

Reply:
Fixed this by adding a `styled` wrapper component for the button actions. I wanted to avoid additional styled components, but this fixes the problem (assuming we do care about button order).